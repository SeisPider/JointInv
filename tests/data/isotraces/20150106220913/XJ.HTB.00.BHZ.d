SURF96 R U X -1 80.00000 3.55332 0.18773
SURF96 R U X -1 75.00000 3.63061 0.18374
SURF96 R U X -1 70.00000 3.75525 0.18347
SURF96 R U X -1 65.00000 3.77326 0.17200
SURF96 R U X -1 60.00000 3.63927 0.14769
SURF96 R U X -1 55.00000 3.68536 0.13884
SURF96 R U X -1 50.00000 3.58791 0.11963
SURF96 R U X -1 48.00000 3.55795 0.11293
SURF96 R U X -1 46.00000 3.54032 0.10716
