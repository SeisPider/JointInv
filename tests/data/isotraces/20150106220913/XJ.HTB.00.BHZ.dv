MFT96 R U T -1 80.314    3.5627    1.0000    5380.4     0.9980E-06   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 78.769    3.5880    1.0000    5380.4     0.1093E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 77.283    3.6109    1.0000    5380.4     0.1185E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 75.852    3.6315    1.0000    5380.4     0.1275E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 74.473    3.6497    1.0000    5380.4     0.1361E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 73.143    3.6657    1.0000    5380.4     0.1442E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 71.860    3.6794    1.0000    5380.4     0.1519E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 70.621    3.6910    1.0000    5380.4     0.1592E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 69.424    3.7004    1.0000    5380.4     0.1661E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 68.267    3.7079    1.0000    5380.4     0.1727E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 67.148    3.7134    1.0000    5380.4     0.1790E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 66.065    3.7170    1.0000    5380.4     0.1851E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 65.016    3.7190    1.0000    5380.4     0.1911E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 64.000    3.7193    1.0000    5380.4     0.1971E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 63.015    3.7180    1.0000    5380.4     0.2032E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 62.061    3.7154    1.0000    5380.4     0.2095E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 61.134    3.7116    1.0000    5380.4     0.2160E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 60.235    3.7065    1.0000    5380.4     0.2229E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 59.362    3.7005    1.0000    5380.4     0.2300E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 58.514    3.6935    1.0000    5380.4     0.2375E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 57.690    3.6858    1.0000    5380.4     0.2454E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 56.889    3.6774    1.0000    5380.4     0.2534E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 56.110    3.6684    1.0000    5380.4     0.2617E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 55.351    3.6591    1.0000    5380.4     0.2700E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 54.613    3.6494    1.0000    5380.4     0.2782E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 53.895    3.6396    1.0000    5380.4     0.2861E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 53.195    3.6296    1.0000    5380.4     0.2936E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 52.513    3.6197    1.0000    5380.4     0.3003E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 51.848    3.6100    1.0000    5380.4     0.3063E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 51.200    3.6005    1.0000    5380.4     0.3111E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 50.568    3.5913    1.0000    5380.4     0.3146E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 49.951    3.5826    1.0000    5380.4     0.3166E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 49.349    3.5744    1.0000    5380.4     0.3170E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 48.762    3.5669    1.0000    5380.4     0.3156E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 48.188    3.5602    1.0000    5380.4     0.3123E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 47.628    3.5542    1.0000    5380.4     0.3071E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 47.080    3.5492    1.0000    5380.4     0.2999E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 46.545    3.5453    1.0000    5380.4     0.2908E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 46.022    3.5425    1.0000    5380.4     0.2798E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 80.000    3.5678    1.0000    5380.4     0.1017E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 75.000    3.6428    1.0000    5380.4     0.1328E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 70.000    3.6959    1.0000    5380.4     0.1628E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 65.000    3.7190    1.0000    5380.4     0.1912E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 60.000    3.7049    1.0000    5380.4     0.2248E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 55.000    3.6545    1.0000    5380.4     0.2739E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 50.000    3.5833    1.0000    5380.4     0.3165E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
MFT96 R U T -1 48.000    3.5582    1.0000    5380.4     0.3106E-05   55.204899  163.244598   44.086899   86.584396 0  1 COMMENT
