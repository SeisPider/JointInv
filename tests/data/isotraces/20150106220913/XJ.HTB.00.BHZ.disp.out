SURF96 R U T -1 80.000    3.5678    1.0000    
SURF96 R C T -1 80.000    6.4060    1.0000    
SURF96 R U T -1 75.000    3.6428    1.0000    
SURF96 R C T -1 75.000    6.1098    1.0000    
SURF96 R U T -1 70.000    3.6959    1.0000    
SURF96 R C T -1 70.000    5.8508    1.0000    
SURF96 R U T -1 65.000    3.7190    1.0000    
SURF96 R C T -1 65.000    5.6191    1.0000    
SURF96 R U T -1 60.000    3.7049    1.0000    
SURF96 R C T -1 60.000    5.4060    1.0000    
SURF96 R U T -1 55.000    3.6545    1.0000    
SURF96 R C T -1 55.000    5.2029    1.0000    
SURF96 R U T -1 50.000    3.5833    1.0000    
SURF96 R C T -1 50.000    5.0037    1.0000    
SURF96 R U T -1 48.000    3.5582    1.0000    
SURF96 R C T -1 48.000    4.9246    1.0000    
